src/org/usfirst/frc2175/subsystem/intake/DreamIntakeSubsystem.java

```diff
@@ -32,12 +35,10 @@ public void setLiftSpeed(double speed) {
     }
 
     public boolean isCompletelyOut() {
-        // TODO Fill in
-        return false;
+        return dreamIntakeDownSwitch.get();
```

Comment:
This code looks fine to me except that the methods are named really strangely. We don't describe the dream's position as "out" and "in" so maybe we should rename this methods `isCompletelyDown()` and `isCompletelyUp()`.